packages/core/src/events/handlers/notifyToClientEvaluationResultCreatedJob.ts

```diff
@@ -10,9 +10,10 @@ export const notifyToClientEvaluationResultCreatedJob = async ({
   data: EvaluationResultCreatedEvent
 }) => {
   const { evaluation, documentLog, evaluationResult } = event.data
-  const { evaluationResultsScope, baseQuery } = createEvaluationResultQuery(
-    evaluation.workspaceId,
-  )
+  // FIXME: DRY. This should be using same query as
+  // computeEvaluationResultsWithMetadata
+  const { evaluationResultsScope, baseQuery } =
+    createEvaluationResultQueryWithErrors(evaluation.workspaceId)
```

Comment:
We have to review this event handlers. I tried to use new `computeEvaluationResultsWithMetadata` but it already includes the `where` clause. But if we solve that we can remove all these 
`createEvaluationResultQueryWithErrors` that are not used anywhere now I think.